cfbs/utils.py

```diff
@@ -280,7 +281,6 @@ def immediate_files(path):
 
 
 def path_append(dir, subdir):
-    dir = os.path.abspath(os.path.expanduser(dir))
```

Comment:
Same as above, and below in `cfengine_dir()`. Avoid changing the implementation, add only tests first, and then introduce a separate PR, with individual commits explaining why the change is necessary and correct.